### PR TITLE
Return correct height in intrinsicContentSize

### PR DESCRIPTION
**Pull request by AlimoRabbani**
Calculate the intrinsic height of the animation view based on height of the image sheet and number of frames.

### CODE REVIEW

**Comment by jverkoey** [comment]
Whoever gets to this first: because we don't have any CI yet please check out the PR locally and build/run the app to verify no regressions.

File: src/MDFSpritedAnimationView.m [hunk 1]
```diff
@@ -80,7 +80,7 @@ - (instancetype)initWithSpriteSheetImage:(UIImage *)spriteSheetImage
 - (CGSize)intrinsicContentSize {
   if (_spriteSheetImage) {
     CGFloat width = _spriteSheetImage.size.width;
-    return CGSizeMake(width, width);
+    return CGSizeMake(width, _spriteSheetImage.size.height / _numberOfFrames);
```
Comment (randallli) [inline src/MDFSpritedAnimationView.m]: can you guard against divide by zero please?

**Review by randallli — DISMISSED** [review]

File: src/MDFSpritedAnimationView.m (hunk 1 above)
Comment (AlimoRabbani) [reply src/MDFSpritedAnimationView.m]: Done, but there are other divisions happening across this files that are not guarded, so I think it's implicitly assumed that numberOfFrames is always greater or equal to 1.

**Comment by AlimoRabbani** [comment]
I made the changes requested.

**Comment by randallli** [comment]
running it locally now

**Comment by randallli** [comment]
Looks like existing use cases are all working. I changed the destination branch to develop. and I will cut a release after i merge.

**Pull request merged**